src/main.js

```diff
@@ -13,21 +15,28 @@ var savedCovers = [
 var currentCover;
 
 // Add your event listeners here 👇
-randomBtn.addEventListener('click', getRandomIndex)
-
+randomBtn.addEventListener('click', randomize)
+makeNewBtn.addEventListener('click', showMakeOwnPage)
 
 // Create your event handlers and other functions here 👇
-// function randomize() {
-  function getRandomIndex(titles) {
-    return Math.floor(Math.random() * titles.length);
-  }
-  function getRandomIndex(covers) {
-    return Math.floor(Math.random() * covers.length);
-  }
-  function getRandomIndex(descriptors) {
-    return Math.floor(Math.random() * descriptors.length);
-  }
-// }
+randomize();
+function randomize() {
+    var newCover = getRandomIndex(covers)
+    var newTitle = getRandomIndex(titles)
+    var newTag1 = getRandomIndex(descriptors)
+    var newTag2 = getRandomIndex(descriptors)
+    coverPic.src = covers[newCover]
+    coverTitle.innerText = titles[newTitle]
+    coverTagLine.innerText = `A tale of ${descriptors[newTag1]} and ${descriptors[newTag2]}`
+    return new Cover(covers[newCover], titles[newTitle], descriptors[newTag1], descriptors[newTag2])
```

Comment:
this return value isn't being used or saved anywhere else yet. let's refactor next so that we save and use that instance of the Cover class